_grades/A9.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: A9 - Lead Software Engineer
```

Comment:
is this title right?

see https://capgemini.github.io/grade-ladder/ for existing set of titles

Reply:
I thought A8 was "Senior Software Engineer" and A9 is "Managing Software Engineer", which of course begs the question, what should the others be!

Reply:
I wonder if it should look like this:
A1 - Apprentice Software Engineer
A2 - Apprentice Software Engineer
A3 - Software Engineer
A4 - Software Engineer
A5 - Lead Software Engineer
A6 - Lead Software Engineer
A7 - Senior Software Engineer
A8 - Senior Software Engineer
A9 - Managing Software Engineer
A10 - Principle Software Engineer

Reply:
Lead and Senior should be the other way around

Reply:
I think what these should be, and what they actually are don't match. Looking at OSCE, "Apprentice Software Engineer" does not exist, as Global Grade A is usually "Software Engineer", Grade B seems to be "Software Engineer Lead", then C is "Senior Software Engineer" and D is "Managing Software Engineer". I believe when we rolled out these job titles, a few years ago now, then Senior was applied to Grade C.

Reply:
So I agree with @chriswilks that I think of a software engineer lead as something like a "tech lead", so would be above a senior software engineer in my view. I was briefly reading about the global grade structure, based off CIS (Cloud Infrastructure Services) who looks to have a similar grade split to us the following matches up with the global grades:

A1 - A3 = Global A
A4 - A6 = Global B
A7 - A8 = Global C
A9 = Global D
A10 = Global E
VP = Global F

See this link on talent = https://talent.capgemini.com/uk/news/16293

Perhaps we should agree on suitable levels then specify them within our grade ladder for future reference?

Reply:
Oh found another link which seems to line up with the above as well: https://talent.capgemini.com/uk/pages/HR/career_development/grade_structure/

Reply:
So disappointing @nighk that you didn't find this on our Confluence's FAQ page..... However the more difficult part, that is less well defined is the job titles

Reply:
Hahaha I may have made that harder for myself than I needed, I was right though :-P

Reply:
@glawrence Looks like you are right about the grade titles by the way: https://talent.capgemini.com/media_library/Medias/Applications_Services_UK/2015_Apps_Career_Framework_Software_Engineer_UK_version.pdf

That was from 2015 though, and I would suggest that doesn't line up with industry standards very well. So I think personally I would propose renaming them to better match peoples expectations. e,.g. I'm not sure someone tagged as a Senior Software Engineer would expect to be leading multiple teams of people and dealing with bids...

So as a different suggestion how about the following:

Reply:
I also agree with Chris, I've always thought the company corporate titles were somewhat backwards. For example if I was to ever be promoted to senior software engineer, I probably wouldn't put it in my CV because it does seem like a back step as the industry sees lead being a role subsequent to senior.

Reply:
Just my two cents on this. But I've always seen the multiple grades sharing same title a bit confusing. To be promoted into the same title seems a bit weird. Perhaps we use unique titles? Or perhaps reduce the amount of grades? Not sure how much of a tree shake that would be that with regards to the corporate structure?

Reply:
I have been in Capgemini for a long time, so have no idea what the cool and on-trend industry terms are... If I remember correctly from when these terms were introduced, then the "prefix part", so Senior, Managing, Principle etc relate to global grade, because the titles were a global job title thing, this is how/why A7&A8 are both senior.

This then brings us to who is the audience? I suspect this ladder is most used by our team, so the titles should align internally to avoid confusion, however this is in conflict with it being available externally and public, but could then lead to confusion if people join.

My conclusion on this is, bit of a mess, needs sorting properly, but that won't happen any time soon

Reply:
As we have a nice progressive number system 1-10 and are unlikely to be able to alter Capgemini's official titles any time soon, how about we leave it at that, ie A1 - Sofware Engineer Level 1, A2 - Software Engineer Level 2 and so on. That way, internal users can just look at the A-numbers, and external users won't be put off by the misleading titles